tests/std/include/range_algorithm_support.hpp

```diff
@@ -69,12 +69,12 @@ namespace test {
 
     template <class T>
     void operator&(T&&) {
-        STATIC_ASSERT(always_false<T>);
+        STATIC_ASSERT(false);

```

Comment:
Should we also mark the definition of `always_false` on line 22 as `TRANSITION, VSO-2016422 (EDG)` to ensure it isn't missed when the bug gets fixed?

Reply:
I pushed a fix for this, and also annotated the definition of `always_false` in `tests/P0898R3_concepts`

Reply:
Great idea, thank you! :heart_eyes_cat: